projects/plugins/wpcomsh/.phan/config.php

```diff
@@ -14,6 +14,9 @@
 	dirname( __DIR__ ),
 	array(
 		'exclude_file_regex'    => array( 'tests/lib/mocks' ),
+		'exclude_file_list'     => array(
```

Comment:
Prevent the Phan warning `PhanRedefinedClassReference` caused by the `Jetpack_Color` being currently defined twice.